### PR TITLE
add dagster project cli to apidocs

### PR DESCRIPTION
**Pull request by yuhan**

### CODE REVIEW

**Comment by yuhan** [comment]
Current dependencies on/for this PR:
* master
  * **PR #9484** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/9484" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/9484?utm_source=stack-comment).

**Review by sryza — APPROVED** [review]
Nice

File: python_modules/dagster/dagster/_cli/project.py [hunk 1]
```diff
@@ -32,6 +32,8 @@ def project_cli():
     "This CLI enables you to quickly bootstrap your project with an officially maintained example."
 )
 
+list_examples_command_help_text = "List the examples that available to bootstrap with."
+
```
Comment (shalabhc) [inline python_modules/dagster/dagster/_cli/project.py:36]: that _are_ available

Comment (shalabhc) [reply python_modules/dagster/dagster/_cli/project.py:36]: Maybe shorten the entire thing to "List available examples for the from-example command."

File: python_modules/dagster/dagster/_cli/project.py [hunk 2]
```diff
@@ -32,6 +32,8 @@ def project_cli():
     "This CLI enables you to quickly bootstrap your project with an officially maintained example."
```
Comment (shalabhc) [inline python_modules/dagster/dagster/_cli/project.py:32]: Outside this review but would be useful to add "See 'list-examples'" or something to help users playing around with the CLI.

**Pull request merged**